invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemsSearch/ItemsSearch.js

```diff
@@ -84,9 +80,7 @@ export class ItemsSearch extends Component {
         <NewButton
           fluid
           text={'New item'}
-          clickHandler={() => {
-            // TODO: EDITOR, implement create form
-          }}
+          clickHandler={goToHandler(BackOfficeRoutes.itemCreate)}
```

Comment:
@topless better a link here? It is a button I see, but what about if I would like to do right-click, open in a new tab? Does this make sense?

Reply:
Actually I agree it should be a link here. I'm not sure what we decided for usability but I like to follow the principle of if it's an action use a button otherwise a link.